punjabi/punjabi2.md

```diff
@@ -0,0 +1 @@
+Click [here](https://en.wikipedia.org/wiki/List_of_Punjabi_people) to know about famous Punjabi people.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.